comm/src/main/scala/coop/rchain/p2p/effects/Communication.scala

```diff
@@ -8,7 +8,7 @@ import coop.rchain.catscontrib._, Catscontrib._
 trait Communication[F[_]] {
   def roundTrip(msg: ProtocolMessage,
                 remote: ProtocolNode,
-                timeout: Duration = Duration(500, MILLISECONDS)): F[CommErr[ProtocolMessage]]
+                timeout: Duration): F[CommErr[ProtocolMessage]]
```

Comment:
please, god no.... `implicit`s only for typeclass machinery and nothing else (maybe few extremes) but not here, especially since `Communication` is a typeclass